README.md

```diff
@@ -20,26 +23,49 @@ require('base16-colorscheme').setup({
 
 *Note: If you don't see colours, try adding `vim.opt.termguicolors = true` to your init.lua*
 
-# Advanced Usage
+## Advanced Usage
+
+To disable highlights for supported plugin(s), call the `with_config` function **before** setting the colorscheme:
 
 ```lua
--- To disable highlights for supported plugin(s), call the `with_config` function **before** setting the colorscheme.
--- These are the defaults.
+-- Call `with_config` **before** setting a colorscheme if you want to tweak anything
 require('base16-colorscheme').with_config({
+    -- These are the defaults.
     telescope = true,
+    telescope_borders = false,
     indentblankline = true,
     notify = true,
     ts_rainbow = true,
     cmp = true,
     illuminate = true,
+    lsp_semantic = true,
+    mini_completion = true,
     dapui = true,
```

Comment:
It'd be great to have some brief descriptions too, e.g:

```lua
    -- These are the defaults.
    telescope = true,          -- Enable theming telescope panels
    telescope_borders = false, -- Also theme telescope panel borders
    indentblankline = true,    -- Set colors for indentblanklins's indentation guides
    notify = true,             -- etc
    ts_rainbow = true,         -- etc
```

I haven't added any because I have no idea what they should be.